modules/cnv_template_wizard_fields_web.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-template-wizard-fields-web]]
+== virtual machine template wizard fields
+
+|===
+|Name |Parameter |Description
+
+|Name
+|
+|Name of the virtual machine template.
+
+|Description
+|
+|Optional description field.
+
+|Namespace
+|
+|Designated namespace in which the template will be created.
+
+.3+|Provision Source
+|PXE
+|Provision virtual machines from PXE menu. Requires a PXE-capable NIC in the cluster.
+
+|URL
+|Provision virtual machines from an image available from a *HTTP* or *S3* endpoint. 
+
+|Container
+|Provision virtual machines from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`
+
+|Operating System
+|
+|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine.
+
+|Flavor
+|small, medium, large, tiny, Custom
+|Presets that determine the amount of CPU and memory allocated to the virtual machines. 
+
+.2+|Workload Profile
```

Comment:
@MarSik not sure it is a good idea to explain it like this. WDYT?

Reply:
Yeah, I would be a bit more verbose with less emphasis on details. The highperformance workload variant will do more than just iothreads in the future (once we have the api for stuff). For example see oVirt - the HP option there disables USB, graphical devices, sound cards, enables iothreads and multiqueues..

Reply:
Thanks for the feedback. Have updated it to:
generic: Virtual machines have a general configuration appropriate for standard usage.
highperformance: Virtual machines have a more efficient configuration optimized for high performance loads.

Reply:
That would be too vague for me..

I am not a tech writer but I would define generic as good compromise between performance and compatibility for the broadest set of usages. HP can then say that some features and compatibility options were sacrificed in favor of better performance of cpu intensive workloads.

Reply:
@MarSik Appreciate it:
generic: A general configuration that balances performance and compatibility for a broad range of workloads.
highperformance: Some features and compatibility options are sacrificed for better performance of CPU-intensive workloads.
However, once we say 'some features and options', it would be good to direct users to something that goes into detail to what those features and options might be. Do we have a resource like that (even if it's an upstream page)? I gather this is likely to change quickly, but still worth including

Reply:
@aburdenthehand  hi, tech writer here. 

I've discussed this with my team, and your suggested `generic` description is great. I suggest also not changing the `highperformance` description. We tend to avoid suggesting sacrifices without in-depth descriptions. 

Everything else looks great in this PR! Will mark peer review done.

Reply:
Cool beans, updated as suggested. Thank you!